index.rb

```diff
@@ -11,7 +12,33 @@
   gem 'nokogiri'
 end
 
-Show = Struct.new(:time, :link, :title, :description, :venue, :keyword_init => true)
+Show = Struct.new(:time, :link, :title, :description, :venue, :keyword_init => true) do
+  def ical_link
+    'data:text/calendar;base64,' +
+    Base64.urlsafe_encode64(
+      [
+        ['BEGIN', 'VCALENDAR'],
+        ['VERSION', '2.0'],
+        ['BEGIN', 'VEVENT'],
+        ['URL', link],
+        ['DTSTART', ical_time(time)],
+        ['DTEND', ical_time(time + (3600 * 3))],
+        ['SUMMARY', title],
+        ['DESCRIPTION', description&.gsub("\n", '\\n')],
```

Comment:
Will this have html in it? Does that matter?

Reply:
I don't think the DESCRIPTION field supports HTML, so if some sneaks in, I don't think it matters.

There might be other characters we need to escape, according to the spec: https://icalendar.org/iCalendar-RFC-5545/3-3-11-text.html

Reply:
> I don't think the DESCRIPTION field supports HTML, so if some sneaks in, I don't think it matters.

It looks like it's breaking for Bottom of the Hill links: https://github.com/davishmcclurg/shows/pull/14